### PR TITLE
Support for guzzle 6.x and 7.x versions

### PR DESCRIPTION
**Pull request by moon-brillar**

### CODE REVIEW

**Comment by moon-brillar** [comment]
[Integrate Laravel Webhook package and backend API](https://trello.com/c/IfdOX7vz/1554-integrate-laravel-webhook-package-and-backend-api)

**Review by FishChen1202 — APPROVED** [review]
r+, thanks!

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "prefer-stable": true,
     "require": {
         "php": ">=8.0",
-        "guzzlehttp/guzzle": "^7.7",
+        "guzzlehttp/guzzle": "^6.3|^7.3",
```
Comment (ericHao22) [inline composer.json:24]: :speech_balloon: comment

或許也可以考慮用v6就好？畢竟v6到v7可能還是會有一些breaking change

**Review by ericHao22 — COMMENTED** [review]
r+ with a comment, thanks!!

**Pull request merged**